src/DotNetWorker.Core/Context/FunctionContextHttpRequestExtensions.cs

```diff
@@ -45,7 +47,7 @@ public static class FunctionContextHttpRequestExtensions
             }
 
             // see output binding entries have a property of type HttpResponseData;
-            var httpOutputBinding = context.GetOutputBindings<HttpResponseData>().FirstOrDefault();
+            var httpOutputBinding = context.GetOutputBindings<HttpResponseData>().First(a => a.BindingType == HttpBindingType);
```

Comment:
`First` will throw if no item matches the predicate.
```suggestion
            var httpOutputBinding = context.GetOutputBindings<HttpResponseData>().FirstOrDefault(a => a.BindingType == HttpBindingType);
```

Reply:
No. It should be FirstOrDefault. Copy paste mistake. This change is not absorbed in Narens other PR as these 2 are PRs are waiting for a broken test fixed in the other PR. Changed to FirstOrDefault in that PR.